tools/validators/instance_validator/tests/fake_instances/GOOD/building_config_with_spaces.yaml

```diff
@@ -20,7 +20,7 @@ US-SEA-BLDG1-GUID:
 CHWS-1-GUID:  
   type: HVAC/CHWS_WDT
   code: CHWS-1
-  cloud_device_id: "foobar"
+  cloud_device_id: "2619178366980754"
```

Comment:
change cloud_device_id to something more generic like 1234567890123456

Reply:
done.